src/main/java/com/esri/core/geometry/ogc/OGCGeometry.java

```diff
@@ -489,7 +496,7 @@ public static OGCGeometry fromEsriShape(ByteBuffer buffer) {
 	}
 
 	public static OGCGeometry fromJson(String string)
-			throws JsonParseException, IOException {
+			throws Exception {
```

Comment:
Just now noticed, this is an interface change which will require changes to callers.